sigstore-java/src/main/java/dev/sigstore/tuf/Updater.java

```diff
@@ -420,7 +420,10 @@ void downloadTargets(Targets targets)
         throw new FileNotFoundException(targetName, fetcher.getSource());
       }
       verifyHashes(entry.getKey(), targetBytes, targetData.getHashes());
-      localStore.storeTargetFile(versionedTargetName, targetBytes);
+
+      // when persisting targets use the targetname without sha512 prefix
```

Comment:
The link you mention suggests checksum-prefixed names for consistent snapshots scheme. Why we should skip checksums here?

Reply:
This is the relevant line of the link:
```
In either case, the client MUST write the file to non-volatile storage as FILENAME.EXT.
```

The code already does fetch by the checksummed prefixed name (`versionedTargetName`), but we write without the prefix.